src/OpenTracing/IScope.cs

```diff
@@ -0,0 +1,95 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="AbstractTag.cs">
```

Comment:
We currently use MIT which is also used by .NET Core so I guess it should be fine. I don't care which license we use but if we want to change it, we need approval from someone. So I'd rather stick to MIT for now since that's what's live on nuget.org 

I'd also prefer not to have license headers in files at all since they are a burden to maintain. Also, implementations like opentracing-go don't use them.

Reply:
Odd, I copied the license from the java files. We release under different licenses?! Sounds like a nightmare :)

Anyway, if you're okay w/o licenses I am definitely too, especially since there's no IP concerns really since it's open source. Will remove

Reply:
fyi: There's an issue regarding different licenses in the specifications-project: https://github.com/opentracing/specification/issues/61

Any discussions about changing the license should happen there and then we'll do whatever the result of that issue is.

Reply:
resolved